delivery_recipe/user_stories.md

```diff
@@ -0,0 +1,39 @@
+# User Stories
+
+What’s your objective? To write your requirements from the point of view of the user, describe them well and be clear about the value and how it fits with the overall project objectives.
+
+User stories are short descriptions of a features from the point of view of the user. The user might be a customer or a member of staff. Although many user stories begin life on a napkin or postit note they will eventually be added to a system such as Rally or Pivotal so that we can share them with our distributed teams.
+
+User stories are generally written as:
+
+* As a [type of user]
+* I want [a thing, a goal]
+* So that [value added]
+
+The reason we write stories in this way is to (a) make sure that stories add real value and (b) the structure helps promote discussion among the team. Those discussions, when we disect stories, question the value, discuss with the actual user and agree an approach to a solution are what drives real quality into our approach.
+
+It’s not vital to use this exact structure, it’s just a guide. If your story is difficult to write in this format then find a way that works for you but remember what’s important when writing your stories.
+
+## Focus on the user
+
+It’s so important to understand the user when writing stories. The reason we write user stories is to solve problems. Generally we write stories when we’ve discovered a problem or we need a new function for a user. For example, it’s reported that there is no facility to reset a password, no way to search for products, no way to book an interview slot or no way to edit an email template.
+
+Who is the user? What is their function? Are they a customer? Are they an admin? Should they really have access to this function?
+
+## Describe the thing
+
+This is the solution that’s going to solve the problem. Describe what you want in detail. What is it you need the story to deliver? Keep the delivery team in mind as you write this.
+
+## Show me an example
+
+You: I want to back up my harddrive
```

Comment:
The example might be more useful if related to a digital service functionality of some kind  (also in the final user story it currently describes archiving rather than backing up ;)  )